### ISSUE TITLE
dkim: consider setting a default MaxVerifications

### ISSUE THREAD

**Issue by emersion** [emersion/go-msgauth#35]
We'll probably want to expose a `DefaultVerifyOptions` variable to allow users to easily customize the defaults (e.g. keeping the default `MaxVerifications` but change the default `LookupTXT`).

**Comment by foxcpp** [comment]
The amount of signatures attached to the message is directly related to the amount of servers that processed it. I believe allowing up to 15 signatures per message is a reasonably safe bet.

**Comment by emersion** [comment]
>The amount of signatures attached to the message is directly related to the amount of servers that processed it.

I don't think so. Only authoritative servers should sign the message (ie. servers which have control over the sender's domain name). Intermediary servers (e.g. mailing lists) shouldn't.

FWIW, OpenDKIM's default is 3.
